R/lintr-deprecated.R

```diff
@@ -2,7 +2,7 @@
 #' @title Deprecated functions in lintr
 #'
 #' @seealso [linters] for a complete list of linters available in lintr.
-#' @evalRd rd_tags("single_quotes_linter")
+#' @evalRd rd_tags(tags = "deprecated")
```

Comment:
Isn't this basically a link to `deprectaed_linters`?

Reply:
True, updated.